src/Components/Scenes/Governance/DomainManagerTable.tsx

```diff
@@ -4,16 +4,24 @@ import TableWrapper from '../../Presentational/Table/TableWrapper';
 import Modal from '../../Presentational/Modal/Modal';
 import {
   deleteDomainOwnersRequestFetch,
-  domainOwnersRequestFetch
+  domainOwnersRequestFetch,
+  addDomainOwnersRequestFetch
 } from '../../../State/Governance/actions';
+import { TextField } from '@material-ui/core';
 
-const DomainManagerTable = (props: any) => {
+const DomainManagerTable = ({
+  isModalOpenAddGovernor,
+  setIsModalOpenAddGovernors
+}: any) => {
   const titleText = `Domain Managers`;
   const columns = [
     { name: 'Name', property: 'owneremail' },
     { name: 'Domain', property: 'domain' }
   ];
   const [isModalOpen, setIsModalOpen] = useState(false);
+  const [domain, setDomain] = useState('');
```

Comment:
same comments i mentioned in pull EDMA-924

can these 3 states be placed in the same object? and also why not place these values in redux as to centralize data, just in case? if not all okay

Reply:
I will be adding in my next PR